sql/core/src/main/scala/org/apache/spark/sql/execution/command/resources.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.hadoop.fs.Path
 
 import org.apache.spark.sql.{Row, SparkSession}
 import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference}
+import org.apache.spark.sql.internal.SQLConf
```

Comment:
Can be removed.

Reply:
Yeahh. I forgot to remove this one. I Will make all the necessary changes.